libs/features/pages/booklets-page/src/lib/components/booklets-hero-section/booklets-hero-section.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+import { HeroText } from '@elewa-website/models/schema/ui/texts';
+import { SliderButtonData } from '@elewa-website/models/schema/ui/buttons';
+
+@Component({
+  selector: 'elewa-website-booklets-hero-section',
+  templateUrl: './booklets-hero-section.component.html',
+  styleUrls: ['./booklets-hero-section.component.scss'],
+})
+export class BookletsHeroSectionComponent {
+  herotexts: HeroText = {
+    title:"Booklets",
+    description:"Your organisation has the potential to turn every task into an opportunity to learn. From growing crops, to nursing someone to good health, to developing the next-generation technology, let’s work together to make the magic happen."
+  }
+
+  buttonData: SliderButtonData = {
+    text: "Unlock your Potential"
+  }
+
```

Comment:
For the purposes of readability and maintainability and keeping the component clean, the 'herotexts' and 'buttonData' variable content can be moved to a Typescript file within the data module and referenced from there.